packages/infrastructure/authentik_atlas_mongodb_sso/README.md

```diff
@@ -0,0 +1,84 @@
+# Atlas MongoDB SSO with Authentik
+
+This module sets up the Authentik Application and Provider for Atlas MongoDB.
+
+## Sample Terragrunt Configuration
+
+```hcl
+include "panfactum" {
+  path   = find_in_parent_folders("panfactum.hcl")
+  expose = true
+}
+
+terraform {
+  source = include.panfactum.locals.pf_stack_source
+}
+
+dependency "authentik_core" {
+  config_path = "../authentik_core_resources"
+}
+
+dependency "kube_authentik" {
+  config_path = "../kube_authentik"
+}
+
+inputs = {
+  acs_url      = "https://auth.mongodb.com/sso/saml2/sample"
+  issuer       = "https://authentik.sample.com"
+  audience     = "https://www.okta.com/saml2/service-provider/sample"
+
+  organization_name   = dependency.authentik_core.outputs.organization_name
+  authentik_namespace = dependency.kube_authentik.outputs.namespace
+  media_configmap     = dependency.kube_authentik.outputs.media_configmap
+  authentik_domain    = dependency.kube_authentik.outputs.domain
+
+  allowed_groups = [
```

Comment:
Our 5 default groups should automatically be propagated. There should be no need to manually specify them.